### DISCUSSION TITLE
z-index in grid field type

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#5989] [Troubleshooting] [answered]
Hi, 

Spotted this today, buttons are hidden.

![Screenshot 2022-05-04 at 16 14 21](https://user-images.githubusercontent.com/72254741/166713515-11191951-ecf0-4160-8dd8-6f41f6ba382e.png)

```
Statamic 3.3.10 Pro
Laravel 9.5.1
PHP 8.0.8
simonhamp/statamic-heroicons 1.0.1
statamic/collaboration 0.4.0
```

**Comment by duncanmcclean** [comment, marked as answer]
This looks like a bug, would you be able to create an issue instead?

**Reply by stephenmeehanuk** [reply]
Done
https://github.com/statamic/cms/issues/5992
